src/pages/components/form/style.mdx

```diff
@@ -83,3 +99,32 @@ content and layout of the design. On mobile, forms can only have one column.
 </div>
 
 <Caption>Structure and spacing measurements</Caption>
+
+## AI presence
+
+The following are the unique styles applied to the component when the AI slug is
+present. Unless specified, all other tokens in the component remain the same as
+the non-AI variant.
```

Comment:
I think an additional blurb about the styling of the components inside the form would be good to add here. Something like:

> The AI styling spec of individual components inside of the form can be found on their respective style tabs.